Changelog.md

```diff
@@ -1,6 +1,14 @@
 ## SDFormat 6.0
 
-### SDFormat 6.X.X (20XX-XX-XX)
+### SDFormat 6.3.1 (2021-07-06)
+
+1. Fix flattening logic for nested model names
+    * [Pull request 597](https://github.com/osrf/sdformat/pull/597)
+
+1. Fix flattening logic for nested model names
```

Comment:
fix title: "Translate poses of nested models inside other nested models"

Reply:
thanks!